snooty.toml

```diff
@@ -4,13 +4,13 @@ intersphinx = ["https://www.mongodb.com/docs/manual/objects.inv"]
 toc_landing_pages = ["/introduction", "/tutorials", "/sink-connector", "/source-connector", "/sink-connector/configuration-properties", "/source-connector/configuration-properties", "/sink-connector/fundamentals", "/source-connector/fundamentals", "/source-connector/usage-examples", "/security-and-authentication", "/troubleshooting"]
 
 [constants]
-mkc = "MongoDB Kafka Connector"
+connector-short = "MongoDB Kafka Connector"
```

Comment:
Suggestion:
I think it could be helpful to either use the established pattern of "avro" (full) and "avro-short" (short) or change the existing constants to match a pattern like "avro-long" (full) and "avro" (short). To me, it doesn't seem necessary to have both "-short" and "-long", especially when there's no "in the middle" version.